ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ptp.schema.yml

```diff
@@ -0,0 +1,163 @@
+            type: str
+      priority1:
+        type: int
+        min: 0
+        max: 255
+        convert_types:
+        - str
+      priority2:
+        type: int
+        min: 0
+        max: 255
+        convert_types:
+        - str
+      ttl:
+        type: int
+        min: 1
+        max: 254
+        description: TTL value
+        convert_types:
+        - str
+      domain:
+        type: int
+        min: 0
+        max: 255
+        convert_types:
+        - str
+      message_type:
+        type: dict
+        keys:
+          general:
+            type: dict
+            keys:
+              dscp:
+                type: int
+                convert_types:
+                - str
+                description: DSCP value
+          event:
+            type: dict
+            keys:
+              dscp:
+                type: int
+                description: DSCP value
+                convert_types:
+                - str
+      monitor:
+        type: dict
+        keys:
+          enabled:
+            type: bool
+            default: true
+          threshold:
+            type: dict
+            keys:
+              offset_from_master:
+                type: int
+                min: 0
+                max: 1000000000
+                description: Offset value
+                convert_types:
+                - str
+              mean_path_delay:
+                type: int
+                min: 0
+                max: 1000000000
+                description: Delay value
+                convert_types:
+                - str
+              drop:
+                type: dict
+                keys:
+                  offset_from_master:
+                    type: int
+                    min: 0
+                    max: 1000000000
+                    description: Offset value
+                    convert_types:
+                    - str
+                  mean_path_delay:
+                    type: int
+                    min: 0
+                    max: 1000000000
+                    description: Delay value
+                    convert_types:
+                    - str
+          missing_message:
+            type: dict
+            keys:
+              intervals:
+                type: dict
+                keys:
+                  announce:
+                    type: int
+                    min: 2
+                    max: 255
+                    convert_types:
+                    - str
+                  follow_up:
+                    type: int
+                    min: 2
+                    max: 255
+                    convert_types:
+                    - str
+                  sync:
+                    type: int
+                    min: 2
+                    max: 255
+                    convert_types:
+                    - str
+              sequence_ids:
+                type: dict
+                keys:
+                  enabled:
+                    type: bool
+                    default: false
```

Comment:
Remove `default: false` as this is implied, and the ptp template will only configure if the value is defined.
```suggestion
```

Reply:
resolved in d18407d